tests/core/focusmanager/editor.js

```diff
@@ -109,7 +109,12 @@ bender.test( {
 			resume( function() {
 				editor.on( 'blur', function() {
 					resume( function() {
-						assert.areSame( 'None', editor.getSelection().getNative().type );
```

Comment:
Are these tests still needed? It seems that now they're checking the native Chrome's implementation.

There is also manual test for it (`core/focusmanager/manual/typinginunfocusedinlineeditor`), which became unneeded.